backpack/extensions/__init__.py

```diff
@@ -28,7 +30,9 @@
     "KFRA",
     "HBP",
     "DiagGGNExact",
+    "BatchDiagGGNExact",
     "DiagGGNMC",
     "DiagGGN",
+    "BatchDiagGGN",
```

Comment:
Since `BatchDiagGGN` is the abstract parent class of `BatchDiagGGNExact` (and in the future `BatchDiagGGNMC`) it may not be required here.